PyPortal_Wakeup_Light/wake_up_light.py

```diff
@@ -0,0 +1,198 @@
+"""
+This example uses a PyPortal and rgbw leds for a simple "wake up" light.
+The strip starts to brighten 30 minutes before set wake up time.
+This program assumes a neopixel strip is attached to D4 on the Adafruit PyPortal.
+"""
+import time
+import board
+import neopixel
+from adafruit_pyportal import PyPortal
+from adafruit_bitmap_font import bitmap_font
+from adafruit_display_text.Label import Label
+
+# Type in time to get up  each day of the week
+default_wake_up = "6:30A"
+up_time_monday = default_wake_up
+up_time_tuesday = default_wake_up
+up_time_wednesday = default_wake_up
+up_time_thursday = default_wake_up
+up_time_friday = default_wake_up
+up_time_saturday = "10:00A"
+up_time_sunday = "10:00A"
+wake_up_times = (up_time_monday,
+                 up_time_tuesday,
+                 up_time_wednesday,
+                 up_time_thursday,
+                 up_time_friday,
+                 up_time_saturday,
+                 up_time_sunday)
+
+# set neopixel min and max brightness
+BRIGHTNESS = 0
+MIN_BRIGHTNESS = 0
+MAX_BRIGHTNESS = 0.85
+#initialize neopixel strip
+num_pixels = 30
+ORDER = neopixel.RGBW
+strip = neopixel.NeoPixel(board.D3, num_pixels, brightness=BRIGHTNESS,
+                          pixel_order=ORDER)
+strip.fill(0)
+# color of strip
+WHITE = (255, 255, 255, 255)
```

Comment:
use just the white portion (0, 0, 0, 255) or you can red/yellow cast it with (64, 0, 0, 255)